packages/near-sdk-js/src/cli/post-install.ts

```diff
@@ -38,14 +38,14 @@ const BINARYEN_VERSION_TAG = `v${BINARYEN_VERSION}`;
 
 const BINARYEN_SYSTEM_NAME =
   PLATFORM === "linux"
-    ? "Linux"
+    ? "linux"
     : PLATFORM === "darwin"
     ? "macOS"
     : PLATFORM === "win32"
     ? "windows"
     : "other";
 
-const BINARYEN_ARCH_NAME = (ARCH == 'aarch64') ? 'ARM64' : ARCH.toUpperCase();
+const BINARYEN_ARCH_NAME = (ARCH == 'aarch64') ? 'ARM64' : ARCH.toLowerCase();
```

Comment:
It does not look right to me.

https://github.com/ailisp/binaryen/releases - the archives are named with Linux and uppercase arch:

binaryen-Linux-ARM64.tar.gz
binaryen-Linux-X64.tar.gz
binaryen-macOS-ARM64.tar.gz
binaryen-macOS-X64.tar.gz

Reply:
correct ... my bad.
Fixed!